features/canvas-2d.yml

```diff
@@ -0,0 +1,68 @@
+name: Canvas 2D rendering context
+description: Canvas 2D provides the 2D rendering context for the drawing surface of a `<canvas>` element.
+spec: https://html.spec.whatwg.org/multipage/canvas.html#2dcontext
+group: canvas
+compat_features:
+  - api.CanvasGradient
+  - api.CanvasGradient.addColorStop
+  - api.CanvasRenderingContext2D
+  - api.CanvasRenderingContext2D.arc
+  - api.CanvasRenderingContext2D.arcTo
+  - api.CanvasRenderingContext2D.beginPath
+  - api.CanvasRenderingContext2D.bezierCurveTo
+  - api.CanvasRenderingContext2D.canvas
+  - api.CanvasRenderingContext2D.clearRect
+  - api.CanvasRenderingContext2D.clip
+  - api.CanvasRenderingContext2D.closePath
+  - api.CanvasRenderingContext2D.createImageData
+  - api.CanvasRenderingContext2D.createLinearGradient
+  - api.CanvasRenderingContext2D.createPattern
+  - api.CanvasRenderingContext2D.createRadialGradient
+  - api.CanvasRenderingContext2D.drawFocusIfNeeded
+  - api.CanvasRenderingContext2D.drawImage
+  - api.CanvasRenderingContext2D.ellipse
+  - api.CanvasRenderingContext2D.fill
+  - api.CanvasRenderingContext2D.fillRect
+  - api.CanvasRenderingContext2D.fillStyle
+  - api.CanvasRenderingContext2D.fillText
```

Comment:
I deleted `canvas-text.yml` and moved the keys from there to here as it's all part of the same feature, and those keys did not change the baseline low/high dates significantly. 

Note that the keys in `canvas-text.yml` are actually generated from the BCD tags. The tool didn't yell at me for having duplicate keys. So hopefully this won't break anything too seriously or cause other headaches.